scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
@bfredl Fixed first argument sounds strange to me, first thought is passing such thing via environment or as a working directory. Or, better be consistent with tests and use paths.lua.in (maybe even moved elsewhere and actually shared with tests).